site/content/en/docs/manual/advanced/analytics-and-monitoring/auto-qa.md

```diff
@@ -78,90 +164,179 @@ To create a **Ground truth** job, do the following:
+This validation mode can use several frame selection methods.
 
-1. Open the task, and find a job in the jobs list.
-2. Click on three dots to open the menu.
-3. From the menu, select **Export annotations**.
+##### Random (the default)
 
-### Delete
+This is a simple method that selects frames into the validation set randomly,
+representing the [basic approach](#basics), described above.
 
-To delete the **Ground truth** job, do the following.
+Parameters:
+- frame count (%) - the percent of the task frames to be used for validation
 
-1. Open the task, and find the **Ground truth** job in the jobs list.
-2. Click on three dots to open the menu.
-3. From the menu, select **Delete**.
+##### Random per job
 
-## Assessing data quality with Ground truth jobs
+This method selects frames into the validation set randomly from each annotation job in the task.
 
-Once you've established the **Ground truth** job, proceed to annotate the dataset.
+It solves one of the issues with the simple Random method that some of the jobs can get
+no validation frames, which makes it impossible to estimate quality in such jobs. Note
+that using this methods can result in increased total size of the validation set.
 
-CVAT will begin the quality comparison between the annotated task and the
-**Ground truth** job in this task once it is finished (on the `acceptance` stage and in the `completed` state).
+Parameters:
+- frame count per job (%) - the percent of the job frames to be used for validation.
+This method uses segment size of the task to select the same number of validation frames
+in each job, if possible.
 
-> **Note** that the process of quality calculation may take up to several hours, depending on
-> the amount of data and labeled objects, and is **not updated immediately** after task updates.
+### Honeypots
 
-To view results go to the **Task** > **Actions** > **View analytics**> **Performance** tab.
+In this mode some random frames of the task are selected into the validation set.
+Then, validation frames are randomly mixed into regular annotation jobs. It can also be
+called "Ground Truth pool", reflecting the way validation frames are used.
+This mode can only be used at task creation and cannot be changed later.
 
-![Add new job](/images/honeypot05.jpg)
+The mode has some limitations on the compatible tasks:
+- It's not possible to use it for an already existing task, the task has to be recreated.
+- This mode assumes random frame ordering, so it is only available for image annotation tasks a
+nd not for ordered sequences like videos.
+- Tracks are not supported in such tasks.
 
-### Quality data
+The validation set can be managed after the task is created - annotations can be edited,
+frames can be excluded and restored, and honeypot frames in the regular jobs can be changed.
+However, it's not possible to select new validation frames after the task is created.
 
-The Analytics page has the following fields:
+Parameters:
```

Comment:
Consider using the same parameter names as used in UI: `Overhead per job` and `Total honeypots`